assets/queries/openAPI/paths_object_empty/query.rego

```diff
@@ -0,0 +1,28 @@
+package Cx
+
+import data.generic.openapi as openAPILib
+
+CxPolicy[result] {
+	doc := input.document[i]
+	openAPILib.checkOpenAPI(doc) != "undefined"
+	object.get(doc, "paths", "undefined") != "undefined"
+
+	checkPathsObject(doc.paths)
+
+	result := {
+		"documentId": doc.id,
+		"searchKey": "openapi.paths",
+		"issueType": "IncorrectValue",
+		"keyExpectedValue": "The Paths Object should should not be empty",
+		"keyActualValue": "The Paths Object is empty",
+	}
+}
+
+checkPathsObject(paths) {
```

Comment:
Should be snake case, since it is a function

Reply:
we should take care of that